### PR TITLE
[risk = no] "cannot fetch in tests" -> StubImplementationRequired

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/testing/stubs/stub-utils.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export const StubImplementationRequired = Error('Cannot fetch in tests.  Please implement stub method.');
```
Comment (jmthibault79) [inline ui/src/testing/stubs/stub-utils.tsx:1]: Or maybe StubRequired / "please stub the method"

Comment (calbach) [inline ui/src/testing/stubs/stub-utils.tsx:1]: This is named like a class, but it's actually a constant, please lower case.

Also, maybe put Error in the name: `stubNotImplementedError`, ?

Comment (calbach) [inline ui/src/testing/stubs/stub-utils.tsx:1]: opt nit: Personally I strongly prefer avoiding double space after periods in comments/error messages, but I realize others may have strongly held preferences in the other direction.

**Review by dolbeew — APPROVED** [review]
Nice change, thanks.

**Comment by calbach** [comment]
FYI the deploy script doesn't pick up on spaces.

**Pull request merged**